### PR TITLE
#165020850 Enable Image Support

### PR DESCRIPTION
**Pull request by kossy360**
#### Description
This PR adds image support by interfacing with cloudinary through the cloudinary SDK for node.js

#### Type of Change
- Chore
  - image upload to cloudinary
  - image deletion from cloudinary
  - add multer and cloudinary as dependencies

#### Pivotal Tracker ID
#165020850

### CODE REVIEW

File: src/helpers/imageControl.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  static deleteImage(url) {
+    return new Promise(
+      (res, rej) => {
+        cloudinary.v2.api.delete_resources(extractId(url),
+          (error, result) => {
+            if (error) return rej(error);
+            res(result);
+          });
+      }
+    );
```
[the remark this answers is not in the document]
Comment (profmcdan) [reply src/helpers/imageControl.js]: Resolve this immediately

[the remark this answers is not in the document]
Comment (profmcdan) [reply src/helpers/imageControl.js]: @kossy360

[the remark this answers is not in the document]
Comment (kossy360) [reply src/helpers/imageControl.js]: ok, on it

**Comment by kossy360** [comment]
violations found by hound CI have been fixed

**Comment by Bobjayafam** [comment]
LGTM

**Comment by Jesse-efe** [comment]
LGTM

**Comment by Joyce-O** [comment]
LGTM

**Review by fuchodeveloper — APPROVED** [review]

**Review by kayroy247 — COMMENTED** [review]
LGTM

**Pull request merged**